html/ybc.hbs

```diff
@@ -0,0 +1,4 @@
+<figure class="figure">
+  <img src="./images/Yellow Brick Code logo.png" class="figure-img img-fluid rounded" alt="Yellow Brick Code">
```

Comment:
Any reason why we want to rename the logo file to have whitespace in its name?
As far as I'm aware of, it's generally a better practice to not use any whitespace in filenames as it's more prone to errors.

Also, if we do decide to updated the logo name, we also need to replace other occurrences where the logo file is called such as in `app.js`

Reply:
This should not have been added to this PR. I did these changes in registration branch and did not push them. I will see if I can remove these